### ISSUE TITLE
comment-or-uncomment-region generates XML comment

### ISSUE THREAD

**Issue by FabriceSalvaire** [Sorixelle/astro-ts-mode#6]
`comment-or-uncomment-region` must generate Javascript comment in the code part.

**Comment by Sorixelle** [comment]
Good catch. I'm not quite sure what the best way to fix this is - I'll have to do some asking around.

**Comment by Sorixelle** [comment]
I've [asked emacs-devel](https://lists.gnu.org/archive/html/emacs-devel/2025-03/msg00454.html) for some assistance on this one - looks like it's going to take some rework in Emacs core to make this possible. I'll stay in the loop on that thread, hopefully something comes out of it soon.
